src/tasks/RhythmTask.py

```
@@ -0,0 +1,293 @@
+        ))
+
+    def _handle_finish(self):
+        """关闭结算界面"""
+        self.log_info("关闭结算界面")
+        self.sleep(1.5)
+        self.click(FINISH_CLOSE_POS[0], FINISH_CLOSE_POS[1])
+        self.sleep(1.0)
+
+    def _handle_restart(self) -> bool:
+        """
+        在选歌界面点击开始演奏，然后等待进入音游。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)